include/keyword_vpd_parser.hpp

```diff
@@ -56,12 +56,13 @@ class KeywordVpdParser : public ParserInterface
      * Finding the 2's complement of sum of all the
      * keywords,values and large resource identifier string.
      *
-     * @param checkSumStart - vpd iterator pointing at checksum start value
-     * @param checkSumEnd - vpd iterator pointing at checksum end value
+     * @param[in] i_checkSumStart - vpd iterator pointing at checksum start
```

Comment:
abbreviations should be in all caps.
VPD at some place is written as vpd at some place it is VPD.
Please have same across file "VPD".